pkg/userop/middleware.go

```diff
@@ -160,6 +161,25 @@ func convertAndSetGasLimits(
 	return nil
 }
 
+// / convertAndOverwriteGasLimits overwrites gas limits with the ones from the estimate.
+func convertAndOverwriteGasLimits(
```

Comment:
I suggest renaming `convertAndOverwriteGasLimits` to `overwriteGasLimits` since `convert` method just parses `big.Int` values into `decimal.Decimal`, which is the thing we do a lot across the lib but still we do not prefix all related function

Reply:
I agree that we can omit the `convert` prefix, which would mean that we can rename `convertAndApplyGasLimits` too. I was trying to think of a name that would imply conditional application of the calculated limits, but was not able to find a suitable name. Maybe you got some ideas?

Reply:
Given the structure of the function, it solely applies estimates after an initial parsing of the values, without any conditional logic. So I believe the current name is appropriate

Reply:
Sorry for not enough context. 

> ... which would mean that we can rename convertAndApplyGasLimits too. I was trying to think of a name that would imply conditional application of the calculated limits, but was not able to find a suitable name. Maybe you got some ideas?

This part was solely about `convertAndApplyGasLimits(...)` function, which is the one above the `convertAndOverwriteGasLimits` in the file.

Reply:
Oh, now I see. I guess `overrideGasLimitsIfUnset` is good enough. It clearly conveys that the function sets the gas limits only if they are not already set.

By the way, both functions can share the prefix to highlight the similarities in their behaviour: `overrideGasLimits` and `overrideGasLimitsIfUnset`

Reply:
Or maybe `overwriteGasLimits` and `overrideGasLimits` respectively. But here we get to English nuances:
- "override" is the cancellation of some previous action or decision;
- "overwrite" specifically refers to something being written over something previously written;

Reply:
> By the way, both functions can share the prefix to highlight the similarities in their behaviour: overwriteGasLimits and overwriteGasLimitsIfUnset.

Hmm, I really like these options. Thanks!

Reply:
b556929cb2aa33c3e2bfe6562817a30cae5ee2e3